GiTinder/Controllers/UsersController.cs

```diff
@@ -45,5 +45,55 @@ public async Task<GeneralApiResponseBody> Login([FromBody] LoginRequestBody logi
             }
             return responseBody;
         }
+
+        [HttpGet("/profile")]
+        public object Profile()
+        {
+            var usertoken = Request.Headers["X-Gitinder-Token"];
+            GeneralApiResponseBody responseBody;
+            var responseProfile = _userServices.FindUserByUserToken(usertoken);
+
+            if (string.IsNullOrEmpty(usertoken))
+            {
+                Response.StatusCode = 403;
+                responseBody = new ErrorResponseBody("Unauthorized request!");
+            }
+            if (_userServices.TokenExists(usertoken))
+            {
+                responseBody = new ProfileResponse(responseProfile.Username, responseProfile.Avatar,responseProfile.Repos);
```

Comment:
@dypher91 repos should be a list of string splitted by the semicolon

Reply:
done